src/components/ClubHome/Daily/DailyControl.js

```diff
@@ -16,81 +16,102 @@ const DailyControl = ({
   setColor,
   getInfo,
 }) => {
-  const onClickModify = () => {
+  let token;
+
+  const onClickModify = (el) => {
     setTitle(el.title);
     setPeriod([el.startDate, el.endDate]);
     setNo(el.no);
     setColor(el.colorCode);
     setPop(3);
   };
+
+  const onDelete = (el) => {
```

Comment:
```suggestion
  const onDeleteSchedule = (el) => {
```
그냥 onDelete는 어떤걸 삭제하는지 한 눈에 알아보기 힘들어요
뒤에 목적어(?)를 붙혀줍시다